packages/ui-react/src/components/modal/ModalStyles.tsx

```diff
@@ -3,6 +3,8 @@ import styled, { css } from '../../utils/styledWrapper.js';
 import { Close, ArrowBack } from '../../icons';
 import { ButtonIcon } from '../button';
 
+const MaxInnerWidth = 600;
```

Comment:
this naming is usually related to classes, please use `maxInnerWidth` or `MAX_INNER_WIDTH` if you want to show this explicitly as const

Reply:
also, is there any reason to not inline this/use as styled-components mixin style?

Reply:
Will rename. I don't want to make it complicated, the variable just works for me and I don't see any issues with it.